### PR TITLE
Add remote servers to server using env variable [10613]

### PR DESCRIPTION
**Pull request by EduPonz**
Merge after #238

### CODE REVIEW

**Comment by richiprosima** [comment]
Build status: [Build Status](http://jenkins.eprosima.com:8080/job/FastRTPS%20Docs%20Manual/768/)

File: docs/fastdds/env_vars/env_vars.rst [hunk 1]
```diff
@@ -62,14 +62,17 @@ For more information about XML profiles, please refer to :ref:`xml_profiles`.
...
-(:ref:`Simple discovery<simple_disc_settings>`).
-For more information on how to configure the discovery mechanism in *Fast DDS*, please refer to :ref:`discovery`.
+    is set to |SIMPLE|, |SERVER|, or |BACKUP|.
+    In any other case, the environment variable has no effect.
+
+Setting this variable configures the :ref:`DomainParticipant<dds_layer_domainParticipant>` to connect to one or more
+*servers* using the :ref:`Discovery Server<discovery_server>` discovery mechanism.
+
+* If ``ROS_DISCOVERY_SERVER`` is defined, and the ``DomainParticipant``'s :ref:`discovery protocol<discovery_protocol>`,
+  is set |SIMPLE|, then Fast DDS will instead configure it as |CLIENT| of the given *server*.
```
Comment (JLBuenoLopez-eProsima) [inline docs/fastdds/env_vars/env_vars.rst:72]:
```suggestion
  is set to |SIMPLE|, then Fast DDS will instead configure it as |CLIENT| of the given *server*.
```

File: docs/03-exports/aliases-api.include [hunk 2]
```diff
@@ -658,6 +658,7 @@
 .. |STATIC| replace:: :cpp:enumerator:`STATIC<rtps::fastrtps::eprosima::DiscoveryProtocol::STATIC>`
 .. |SERVER| replace:: :cpp:enumerator:`SERVER<rtps::fastrtps::eprosima::DiscoveryProtocol::SERVER>`
 .. |CLIENT| replace:: :cpp:enumerator:`CLIENT<rtps::fastrtps::eprosima::DiscoveryProtocol::CLIENT>`
+.. |CLIENTS| replace:: :cpp:enumerator:`CLIENT<rtps::fastrtps::eprosima::DiscoveryProtocol::CLIENT>`
```
Comment (JLBuenoLopez-eProsima) [inline docs/03-exports/aliases-api.include:661]: Even though you define this new tag, the printed page keeps showing `CLIENT`. Therefore this addition does not apply as expected and it is unnecessary.

**Review by JLBuenoLopez-eProsima — APPROVED** [review]
LGTM. A couple of nitpicks.

**Review by JLBuenoLopez-eProsima — APPROVED** [review]
LGTM

**Comment by richiprosima** [comment]
Build status: [Build Status](http://jenkins.eprosima.com:8080/job/FastRTPS%20Docs%20Manual/769/)

**Pull request merged**